test/support/postgresql/import_examples.rb

```diff
@@ -17,5 +17,83 @@ def should_support_postgresql_import_functionality
         assert_equal 1, result.num_inserts
       end
     end
+
+    describe "importing objects with ids subobjects" do
```

Comment:
The only thing I don't like about `subobjects` is that we don't accept generic objects. We only accept associated ActiveRecord models which have already been named for us: associations. I feel that if we use use that name it will be more relevant and intuitive to future readers of this code (including the three of us).